Makefile

```diff
@@ -335,7 +335,7 @@ create-cluster: $(CLUSTERCTL) $(KUSTOMIZE) $(ENVSUBST) ## Create a development K
 
 	# (Re-)install Core providers
 	$(CLUSTERCTL) delete --all
-	$(CLUSTERCTL) init --core cluster-api:v0.3.3 --bootstrap kubeadm:v0.3.3 --control-plane kubeadm:v0.3.3
+	$(CLUSTERCTL) init --core cluster-api:v0.3.5 --bootstrap kubeadm:v0.3.5 --control-plane kubeadm:v0.3.5
```

Comment:
we also have to update go.mod to 0.3.5

Reply:
We should also check if hack/tools/go uses the correct version of controller-tools and cluster-api/hack/tools (compare with CAPA when it already uses 0.3.5)

Reply:
@sbueringer I feel I have already updated the go.mod to 0.3.5 and also the corresponding controller tools in hack/tools. Would you mind elaborating a bit more?

Thanks :)

Reply:
Sorry somehow missed :). The only thing I'm not sure about is cluster-api/hack/tools. Is that one also already the correct one?

Reply:
@sbueringer I have updated the controller tools version to v0.2.9 in hack/tools. I think that is the correct version. I checked with CAPA repo which too uses that version.

Reply:
I was referring to the dependency with that name :)
See changes in https://github.com/kubernetes-sigs/cluster-api-provider-aws/commit/0580fa23476bbd797ecb5b3d1be218077f801662 (to hack/tools/go.mod)

They also upgraded to:
* k8s.io/code-generator v0.18.0
* sigs.k8s.io/cluster-api/hack/tools v0.0.0-20200430151019-59a696e07b5e

Reply:
@sbueringer Ohh okay. Ya, I feel that needs to be updated too.